packer/variables.pkr.hcl

```diff
@@ -88,3 +89,25 @@ variable "version" {
   type        = string
   description = "The semantic version number for the AMI; the version string will be appended automatically to the name tags added to the resulting AMI and snapshot(s)"
 }
+
+variable "artifactory_version" {
+  type        = string
+  default     = "7.41.12"
+  description = "The semantic version number for the Artifactory release"
+}
+
+variable "resource_bucket_name" {
+  type        = string
+  description = "The name of the S3 resources bucket"
+}
+
+variable "jfrog" {
```

Comment:
This needs to be more descriptive in line with the description. Something like  `jfrog_release_url`